src/services/telemetry.ts

```diff
@@ -0,0 +1,156 @@
+import { Request, Response, NextFunction } from "express"
+import { v4 } from 'uuid';
+import _ from 'lodash';
+import { config } from "../configs/Config";
+import axios from 'axios';
+
+export enum OperationType {
+    CREATE = 1,
+    UPDATE
+}
+
+const getDefaults = () => {
+    return {
+        eid: 'AUDIT',
+        ets: Date.now(),
+        ver: '1.0.0',
+        mid: v4(),
+        actor: {
+            id: v4(),
```

Comment:
We might be required to refer the authenticated proper user instead of generating the random UUID all the time.